libbeat/autodiscover/autodiscover.go

```diff
@@ -197,33 +212,20 @@ func (a *Autodiscover) handleStart(event bus.Event) bool {
 func (a *Autodiscover) handleStop(event bus.Event) bool {
 	var updated bool
 
-	configs, err := a.adapter.CreateConfig(event)
-	if err != nil {
-		logp.Debug(debugK, "Could not generate config from event %v: %v", event, err)
+	logp.Debug(debugK, "Got a stop event: %v", event)
+	eventID := getID(event)
+	if eventID == "" {
+		logp.Err("Event didn't provide instance id, ignoring it")
```

Comment:
Can we log some information about the event to make it easier to debug without having to enable debug level?